### DISCUSSION TITLE
way to handle all method using export function all() or something

### DISCUSSION THREAD

**Discussion by sharmapukar217** [rakkasjs/rakkasjs#67] [General]
is there any existing method to handle all http method in a single place like export function all() {
 return new Response("hi, I can be accessed from any http method");
}

**Comment by cyco130** [comment]
It works exactly like you describe, the name of the catch all function is `all`. I though it was documented, turns out it wasn't :)

**Reply by sharmapukar217** [reply]
> It works exactly like you describe, the name of the catch all function is `all`. I though it was documented, turns out it wasn't :)

Really?? I once tried and that was not working.. I guess I missed something then. I'll check it out again. TYSM

**Reply by cyco130** [reply]
If it's not working, it's a bug. Just remember individual methods (get, post etc.) have precedence.

**Comment by sharmapukar217** [comment]
oh i got it. Its working. What I did was default exported instead of named export .
